corehq/apps/reports/view_helpers.py

```diff
@@ -76,52 +76,12 @@ def _index_to_context(index, is_ancestor):
     }
 
 
-def normalize_date(val):
```

Comment:
You also need to remove https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/tests/test_tags.py, right?

Reply:
oh yes, good find, thanks, must've forgotten about that.  Hoping that's the cause of the build failures, since they pass now locally.